spec/integration/runner/variant-reassignment-example-7.spec.js

```diff
@@ -0,0 +1,102 @@
+ * |                                         | Product:                 |                    | Product:                                                     |
+ * |                                         | id: "2"                  |                    | id: "2"                                                      |
+ * |                                         | slug: { de: "produkte" } |                    | slug: { de: "produkte_${timestamp}", _ctsd: "${timestamp}" } |
+ * |                                         | product-type: "pt1"      |                    | product-type: "pt1"                                          |
+ * |                                         | masterVariant: v3        |                    | masterVariant: v4                                            |
+ * |                                         | variants: v4             |                    |                                                              |
+ * +-----------------------------------------+--------------------------+--------------------+--------------------------------------------------------------+
+ * |                                         |                          |                    | Product:                                                     |
+ * |                                         |                          |                    | id: "3"                                                      |
+ * |                                         |                          |                    | slug: { en: "produkte_${timestamp}", _ctsd: "${timestamp}" } |
+ * |                                         |                          |                    | product-type: "pt1"                                          |
+ * |                                         |                          |                    | variants: v2                                                 |
+ * +-----------------------------------------+--------------------------+--------------------+--------------------------------------------------------------+
+ */
+/* eslint-enable max-len */
+describe('Variant reassignment', () => {
+  const logger = utils.createLogger(__filename)
+  let ctpClient
+  let product1
+  let product2
+
+  before(async () => {
+    ctpClient = await utils.createClient()
+    const productType = await utils.ensureProductType(ctpClient)
+    const productDraft1 = utils.generateProduct(['1', '2'], productType.id)
+    productDraft1.slug.en = 'product'
+    product1 = await utils.ensureResource(ctpClient.products, productDraft1)
+
+    const productDraft2 = utils.generateProduct(['3', '4'], productType.id)
+    productDraft2.slug.de = 'produkte'
+    product2 = await utils.ensureResource(ctpClient.products, productDraft2)
+  })
+
+  after(() =>
+    utils.deleteResourcesAll(ctpClient, logger)
+  )
+
+  it('merge variants v1 and v3 + remove variants v2 and v4', async () => {
```

Comment:
It looks like the test in `spec/integration/runner/variant-reassignment-example-6.spec.js` is nearly the same - it merges products from different productTypes so maybe this one can be removed?

Reply:
Yes, it's almost the same, but original products has different product types. We want to check if it will select the right product to update. Maybe this could be done in unit test. I will add TODO here and remove when we finish if test looks redundant.

Reply:
https://github.com/commercetools/commercetools-node-variant-reassignment/pull/8/commits/0673979e77ec274033779eebc688c5c470a8123b